modules/collection/programs/hyprpanel.nix

```
@@ -8,9 +8,10 @@
   ...
 }: let
   inherit (lib.modules) mkIf;
-  inherit (lib.options) mkEnableOption mkPackageOption mkOption;
+  inherit (lib.options) mkEnableOption mkPackageOption mkOption optionalAttrs;
```

`optionalAttrs` Is unused
```suggestion
  inherit (lib.options) mkEnableOption mkPackageOption mkOption;
```